Python/legacy_tracing.c

```diff
@@ -0,0 +1,547 @@
+/* Support for legacy tracing on top of PEP 669 instrumentation
+ * Provides callables to forward PEP 669 events to legacy events.
+ */
+
+#include <stddef.h>
+#include "Python.h"
+#include "opcode.h"
+#include "pycore_ceval.h"
+#include "pycore_instruments.h"
+#include "pycore_object.h"
+#include "pycore_pyerrors.h"
+#include "pycore_pymem.h"
+#include "pycore_pystate.h"       // _PyInterpreterState_GET()
+#include "pycore_sysmodule.h"
```

Comment:
```suggestion
#include <stddef.h>
#include "Python.h"
#include "pycore_ceval.h"
#include "pycore_object.h"
#include "pycore_sysmodule.h"
```
Unused include directives